src/lib/util/cdrom.cpp

```diff
@@ -2285,30 +2320,37 @@ std::error_condition cdrom_file::parse_cue(std::string_view tocfname, toc &outto
 				/* next token on the line is the track type */
 				TOKENIZE
 
-				if (wavlen != 0)
-				{
-					outtoc.tracks[trknum].trktype = CD_TRACK_AUDIO;
-					outtoc.tracks[trknum].frames = wavlen/2352;
-					outinfo.track[trknum].offset = wavoffs;
-					wavoffs = wavlen = 0;
-				}
-				else
-				{
-					outtoc.tracks[trknum].trktype = CD_TRACK_MODE1;
-					outtoc.tracks[trknum].datasize = 0;
-					outinfo.track[trknum].offset = 0;
-				}
 				outtoc.tracks[trknum].subtype = CD_SUB_NONE;
 				outtoc.tracks[trknum].subsize = 0;
 				outtoc.tracks[trknum].pgsub = CD_SUB_NONE;
 				outtoc.tracks[trknum].pregap = 0;
 				outtoc.tracks[trknum].padframes = 0;
+				outtoc.tracks[trknum].datasize = 0;
+				outtoc.tracks[trknum].multicuearea = is_gdrom ? current_area : 0;
 				outinfo.track[trknum].idx0offs = -1;
 				outinfo.track[trknum].idx1offs = 0;
+				outinfo.track[trknum].offset = 0;
+
+				if (wavlen != 0)
+				{
+					outtoc.tracks[trknum].frames = wavlen/2352;
+					outinfo.track[trknum].offset = wavoffs;
+					wavoffs = wavlen = 0;
+				}
```

Comment:
This originally set `outtoc.tracks[trknum].trktype = CD_TRACK_AUDIO;` but directly below this `convert_type_string_to_track_info` gets called which will set `outtoc.tracks[trknum].trktype` based on the TRACK's typestring so I removed it. 

Maybe it would be a better idea to move the call to `convert_type_string_to_track_info` before this so the track type can be forced when a WAVE file type is selected as input?